src/components/Toolbar.js

```diff
@@ -305,9 +305,6 @@ export default class Toolbar extends Component {
         className={toolbarClass}
         style={this.state.position}
         ref="toolbarWrapper"
-        onMouseDown={e => {
-          e.preventDefault();
-        }}
```

Comment:
I think this might be worth further explanation.  Why is this code here in the first place, and why are we okay with getting rid of it?  `onMouseDown` != `onClick` for the record, but that might be a red herring; not sure if this code swallows the onclick too? ¯\_(ツ)_/¯

Reply:
added the further explanation to the commit message

Reply:
I do not believe this was serving any purpose beyond removing the active state highlighting of the button.

Reply:
Ok so I changed the solution to continue to prevent the default if the event target is not an input.